journal-of-the-botanical-research-institute-of-texas.csl

```diff
@@ -0,0 +1,639 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="display-and-sort" page-range-format="chicago">
+  <info>
+    <title>Journal of the Botanical Research Institute of Texas</title>
+    <title-short>JBRIT</title-short>
+    <id>https://csl.mendeley.com/styles/589017421/JBRIT</id>
+    <link href="http://www.zotero.org/styles/chicago-author-date" rel="self"/>
```

Comment:
copy this line and make this one your template (rel="template") and then use this link as rel="self" http://www.zotero.org/styles/journal-of-the-botanical-research-institute-of-texas